<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteEditorClass</class>
 <widget class="QMainWindow" name="SpriteEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>794</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>SpriteEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SelectPolygonID</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_selectPolygonID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>511</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_selectColor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>色を選択</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SetGroupID</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_selectGroupID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>511</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSavePolygonData"/>
    <addaction name="actionSaveOriginalFile"/>
    <addaction name="actionOpenFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_ImageList">
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>600</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);</string>
   </property>
   <property name="windowTitle">
    <string>Image Lists</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="DragListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>261</width>
       <height>611</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>600</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);</string>
   </property>
   <property name="windowTitle">
    <string>ImageViewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="GlWidget" name="OpenGLWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>650</width>
       <height>650</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(152, 255, 114);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ClearPoint">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>670</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>画像ファイルを開く</string>
   </property>
  </action>
  <action name="actionSaveOriginalFile">
   <property name="text">
    <string>ファイルの保存</string>
   </property>
  </action>
  <action name="actionSavePolygonData">
   <property name="text">
    <string>ポリゴンデータの保存</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GlWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DragListWidget</class>
   <extends>QListWidget</extends>
   <header>draglistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="spriteeditor.qrc"/>
 </resources>
 <connections/>
</ui>
